pkg/deployment/agent.go

```diff
@@ -140,6 +140,17 @@ func (a *Agent) Get() *appsv1.DaemonSet {
 								Name:          "admin-http",
 							},
 						},
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path: "/",
+									Port: intstr.FromInt(int(adminPort)),
+								},
+							},
+							InitialDelaySeconds: 1,
+							PeriodSeconds:       15,
+							FailureThreshold:    5,
```

Comment:
Why not the default 3? Just want to understand if these values have been selected for a particular reason. Generally better to fail fast - with the current values the failure wouldn't be dealt with for an additional 30 seconds.

Reply:
I thought 5 would be better for waiting longer before failing.I personally prefer longer liveliness probes, so it would be the last resort if application is not responsive for longer(75s here), specially shorter times could result in cascading failures under high load.